src/style.css

```diff
@@ -0,0 +1,33 @@
+* {
+  margin: 0;
+}
```

Comment:
Don't use * selector for zeroing out your margins or paddings. It's still inefficient for browser to read your web document
Move each BEM block to a separate file
It is a default value